skywalking-sniffer/skywalking-api/src/main/java/com/a/eye/skywalking/plugin/interceptor/enhance/ClassEnhancePluginDefine.java

```diff
@@ -70,8 +69,15 @@
             for (ConstructorInterceptPoint constructorInterceptPoint : constructorInterceptPoints) {
                 newClassBuilder = newClassBuilder.constructor(constructorInterceptPoint.getConstructorMatcher())
```

Comment:
@raphw, is there other way to init a attribute, which is type of EnhancedClassInstanceContext? Maybe not use a custom interception?